.test-infra/jenkins/job_PostCommit_Python_Chicago_Taxi_Example_Dataflow.groovy

```diff
@@ -19,46 +19,44 @@
 import CommonJobProperties as commonJobProperties
 import PostcommitJobBuilder
 import CronJobBuilder
+import LoadTestsBuilder
 
-
-// This job runs the Chicago Taxi Example script on Dataflow
-PostcommitJobBuilder.postCommitJob(
-    'beam_PostCommit_Python_Chicago_Taxi_Dataflow',
-    'Run Chicago Taxi on Dataflow',
-    'Google Cloud Dataflow Runner Chicago Taxi Example',
-    this
-) {
-    description('Runs the Chicago Taxi Example on the Dataflow runner.')
+def chicagoTaxiJob = { scope ->
+    scope.description('Runs the Chicago Taxi Example on the Dataflow runner.')
 
     // Set common parameters.
-    commonJobProperties.setTopLevelMainJobProperties(delegate)
+    commonJobProperties.setTopLevelMainJobProperties(scope)
+
+    def pipelineOptions = [
+        num_workers          : 5,
+        autoscaling_algorithm: 'NONE',
+    ]
 
     // Gradle goals for this job.
-    steps {
+    scope.steps {
         gradle {
             rootBuildScriptDir(commonJobProperties.checkoutDir)
+            commonJobProperties.setGradleSwitches(delegate)
             tasks(':sdks:python:test-suites:dataflow:py2:chicagoTaxiExample')
```

Comment:
Can we switch this to Py3?

Reply:
There's a jira for this: https://issues.apache.org/jira/browse/BEAM-9154. As far as I remember there was something that prevented us from switching to Py3. Not sure if it's still relevant.

Reply:
We should move this suite to Py3 as we are actively discussing dropping Py2 support, and these suites will stop working.